pims/pyav_reader.py

```diff
@@ -22,12 +23,12 @@ def available():
     return av is not None
 
 
-def _to_nd_array(frame):
-    if frame.format.name != 'rgb24':
-        frame = frame.reformat(format="rgb24")
-    frame_arr = np.frombuffer(frame.planes[0], np.uint8)
-    frame_arr.shape = (frame.height, frame.width, -1)
-    return frame_arr
+def _next_video_packet(container_iter):
```

Comment:
This appears to fix a bug I also encountered (and tried to fix in a much less elegant way).